File: a058(mod3)/C++/mod3.cpp
#include <iostream>

int main(void) {
  int amount;
  while(std::cin >> amount) {
    int correspond[3] = {0};
    for(int i = 0; i < amount; i++) {
      int number;
      std::cin >> number;
      correspond[number % 3]++;
    }
    std::cout << correspond[0] << " " << correspond[1] << " " << correspond[2] << '\n';
  }

  return 0;
}

File: a058(mod3)/Python/mod3.py

while True:
    try:
        amount = int(input())
        correspond = [0, 0, 0] # 3k, 3k + 1, 3k + 2
        for i in range(amount):
            number = int(input())
            correspond[number % 3] += 1
        print(f'{correspond[0]} {correspond[1]} {correspond[2]}')
    except:
        break
